engine/src/main/resources/credentials/password.json

```diff
@@ -0,0 +1,20 @@
+{
+	"typeId": "password",
+	"name": "sys:password",
+	"readOnly": true,
+	"configuration": "{\"minLength\":10,\"historySize\":3,\"minClassesNum\":3,\"maxAge\":63058176000,\"denySequences\":true,\"allowLegacy\":false,\"scryptParams\":{\"workFactor\":17,\"blockSize\":8,\"parallelization\":1,\"length\":64},\"resetSettings\":{\"enable\":false}}",
```

Comment:
after thinking - please disable mandatory password expiration - by default without this turned on